### PR TITLE
Add DOI citation badge

### PR DESCRIPTION
**Pull request by mgovers**
Add DOI citation to latest version in https://zenodo.org/account/settings/github/repository/PowerGridModel/power-grid-model# (example: https://zenodo.org/record/8054430 )

The `CITATION.cff` also adds a button to the github repo:

image

Example bibtex generated from the `Citation` part:

```bibtex
@software{Power_Grid_Model_and_Xiang_power-grid-model,
author = {{Power Grid Model} and Xiang, Yu and Salemink, Peter and Bharambe, Nitish and Jagutis, Laurynas and Wang, Chenguang and van den Bogaard, Jonas and Govers, Martinus and Stoeller, Bram and Datta, Sattama and Baaijen, Thijs},
doi = {10.5281/zenodo.8054429},
title = {{power-grid-model}},
url = {https://github.com/PowerGridModel/power-grid-model}
}
```

Example APA generated by that button

```md
Power Grid Model, Xiang, Y., Salemink, P., Bharambe, N., Jagutis, L., Wang, C., van den Bogaard, J., Govers, M., Stoeller, B., Datta, S., & Baaijen, T. power-grid-model [Computer software]. https://doi.org/10.5281/zenodo.8054429
```

### CODE REVIEW

File: CITATION.cff
Comment (TonyXiang8787) [reply CITATION.cff:7]: Which parts have to be updated mannually?

Comment (mgovers) [reply CITATION.cff:7]: let's say there is a new author in the future (e.g. a new team member or contributor), then we would have to manually add it to this list, or create a Github action that verifies that this is still correct.

alternatively, we do not have a `CITATION.cff` file. Then `Zenodo` will automatically update the list of authors, but then you do not get the github `cite this` button.

i'm fine either way... your call

File: README.md
Comment (TonyXiang8787) [inline README.md:85]: Please paste the complete BibTex in here. Also please use the wording like

> If you are using Power Grid Model in your research work, please consider citing our library using the following references.

Comment (TonyXiang8787) [reply README.md:85]: Also add the same text in the introduction of the readthedocs

Comment (mgovers) [reply README.md:85]: will do 🙂

File: CITATION.cff
Comment (mgovers) [reply CITATION.cff:7]: ok :+1:

File: README.md
Comment (TonyXiang8787) [reply README.md:85]: In the BibTex block please add two references: the GitHub and the CIRED paper. I will provide you the CIRED paper BibTex later.

Comment (mgovers) [reply README.md:85]: i will have a look at how to do that and also try to add it to `CITATION.cff`

File: CITATION.cff
Comment (mgovers) [reply CITATION.cff:7]: @TonyXiang8787 is it correct that there are only 3 maintainers listed in the PGM project and that chenguang is one of you? https://github.com/orgs/PowerGridModel/teams/pgm-maintainers

Comment (mgovers) [reply CITATION.cff:7]: there are child team members (whatever that may mean) but chenguang is not in that child team member list

Comment (TonyXiang8787) [reply CITATION.cff:7]: All admins are also maintainers

File: README.md
Comment (TonyXiang8787) [reply README.md:85]: @mgovers  You don't have to embed the CIRED paper in `CITATION.cff`. I am more talking about put the CIRED paper in BibTex in Readme and readthedocs.

File: CITATION.cff
Comment (mgovers) [reply CITATION.cff:7]: right, github is just doing a poor job at showing only the root maintainer members instead of the admin members

do you think it's a good idea to split the PGM_admins child team away from the child team so that outside collaborators can see that there are maintainers and admins in https://github.com/orgs/PowerGridModel/teams

image
image

Comment (TonyXiang8787) [reply CITATION.cff:7]: Let's keep this as separate issue.

File: README.md
Comment (TonyXiang8787) [reply README.md:85]: BibTex for CIRED paper

```bibtex
@inproceedings{Xiang2023,
author = {Xiang, Yu and Salemink, Peter and Stoeller, Bram and Bharambe, Nitish and van Westering, Werner},
booktitle = {CIRED 2023 - The 27th International Conference and Exhibition on Electricity Distribution},
title = {Power grid model: A high-performance distribution grid calculation library},
year = {2023},
volume={2023},
number = {},
pages={1-5}
}
```

File: CITATION.cff
Comment (mgovers) [reply CITATION.cff:7]: on that same note: i see that laurynas is not part of the maintainers nor admins

Comment (TonyXiang8787) [reply CITATION.cff:7]: @mgovers I will add him as maintainer.

File: README.md
Comment (TonyXiang8787) [inline README.md:89]: I don't think we need two heading here. Just make a big Bibtext block with two references.

Also add the same thing in readthedocs (either in index or as a separate page

Comment (mgovers) [reply README.md:89]: will do

in the mean time, i'm still debugging. for some reason, the zenodo publishing using the latest CITATION.cff failed. it is successful using the Github button, though.

**Review by TonyXiang8787 — APPROVED** [review]

**Pull request merged**
